library/packages/src/lib/y2packager/product_reader.rb

```diff
@@ -168,14 +179,14 @@ def product_package(name, _repo_id = nil)
     # read the available products, remove potential duplicates
     # @return [Array<Hash>] pkg-bindings data structure
```

Comment:
Please, adjust the `@return` class (`Array<Y2Packager::Resolvable>`). Actually, that makes me think whether we could transform these resolvable into `Product` instances at this point (or if we are going to miss some information later).

If that's not the case, I am not sure about keeping the `*_products` suffix when we are working with resolvables. It is confusing.